### PR TITLE
Text for TD TF and Binding page

### PR DESCRIPTION
**Pull request by sebastiankb**

### CODE REVIEW

File: docs/activities/tf-td/index.html [hunk 1]
```diff
@@ -4,8 +4,59 @@
...
+		<p>The WoT Thing Description task force is responsible for 
+		defining the information model for the Things, its interpretation and
+		common representation. In addition, the task force covers WoT Binding Templates 
+		topics to define the mapping to concrete IoT protocols and payload encodings.		
+		</p>
+		<div>
+		<h3>Normative Deliverables</h3>
+			<p>Current work is focused on a normative deliverables (WoT Thing Description 1.1) and on a informative deliverables (WoT Binding Templates).
+			</p>
+			<div>
```
Comment (danielpeintner) [inline docs/activities/tf-td/index.html:17]: I suggest to have a `<h3>` for normative deliverables were one can find the TD document and afterwards another `<h3>` for non-normative deliverables were we put bindings templates.

File: docs/activities/tf-td/index.html [hunk 2]
```diff
@@ -4,8 +4,59 @@
...
+				Victor Charpenay (Siemens AG)
+			</dd>
+			<dt>WoT Binding Templates:</dt><dd>
+				Michael Koster (SmartThings),
+				Ege Korkan (Siemens AG) 
+			</dd>
+		</dl>
+		</div>
+		<div>
+			<h3>Resources</h3>
```
Comment (danielpeintner) [inline docs/activities/tf-td/index.html:55]: Shall we add links to the following 2 documents also?
* https://www.w3.org/TR/2020/WD-wot-thing-description11-20201124/
* https://www.w3.org/TR/2020/NOTE-wot-binding-templates-20200130/

**Review by danielpeintner — COMMENTED** [review]
Find below some comments

File: docs/activities/tf-td/index.html (hunk 2 above)
Comment (sebastiankb) [reply docs/activities/tf-td/index.html:55]: Discovery and Architecture have not this information.

File: docs/activities/tf-td/index.html (hunk 1 above)
Comment (sebastiankb) [reply docs/activities/tf-td/index.html]: done

**Comment by egekorkan** [comment]
I have found at https://www.w3.org/WoT/WG/ that binding templates is a separate TF. Maybe it makes sense to merge it into TD TF. In the beginning, it also had its own call but there is no need at the moment.

**Comment by sebastiankb** [comment]
from today's marketing call: We will merge this PR

**Pull request merged**